### ISSUE TITLE
[BUG] Workflow dependency definition does not work, tasks are executed in parallel, pretask field is empty (工作流依赖定义不工作，任务并行执行 pretask字段为空)

### ISSUE THREAD

**Issue by hua6** [apache/incubator-dolphinscheduler#3012]
*For better global communication, please give priority to using English description, thx! *

**Describe the bug**

**To Reproduce**
Steps to reproduce the behavior, for example:

**Expected behavior**

**Screenshots**
![image](https://user-images.githubusercontent.com/33834062/84989841-dcfc4900-b176-11ea-8449-91fb0d2c85be.png)
![image](https://user-images.githubusercontent.com/33834062/84989852-e2599380-b176-11ea-943f-aefb0b56fd46.png)
![image](https://user-images.githubusercontent.com/33834062/84989868-e7b6de00-b176-11ea-89be-e2759a4fb8bd.png)

**Which version of Dolphin Scheduler:**
 -[1.3.1-preview]

**Additional context**

**Requirement or improvement
- Please describe about your requirements or improvement suggestions.

**Comment by hua6** [comment]
git branch is 1.3.0-release， docker build version is 1.3.1

**Comment by xingchun-chen** [comment]
1.There is only one connection between tasks, please clear your browser cache
2.Workflow dependency definition does not execute, please describe the problem in detail
3.Please update the title in English

**Comment by hua6** [comment]
![image](https://user-images.githubusercontent.com/33834062/85002637-b98dca00-b187-11ea-9113-0dc74876a1e1.png)
@xingchun-chen after clean cache,it alse like this 
and export job json 


[{"processDefinitionConnects":"[{\"endPointSourceId\":\"tasks-9149\",\"endPointTargetId\":\"tasks-56829\"},{\"endPointSourceId\":\"tasks-9149\",\"endPointTargetId\":\"tasks-56829\"},{\"endPointSourceId\":\"tasks-56829\",\"endPointTargetId\":\"tasks-45354\"},{\"endPointSourceId\":\"tasks-56829\",\"endPointTargetId\":\"tasks-45354\"},{\"endPointSourceId\":\"tasks-56829\",\"endPointTargetId\":\"tasks-45354\"}]","processDefinitionJson":"{\"tenantId\":1,\"globalParams\":[],\"tasks\":[{\"conditionResult\":{\"successNode\":[\"\"],\"failedNode\":[\"\"]},\"description\":\"\",\"runFlag\":\"NORMAL\",\"type\":\"PYTHON\",\"params\":{\"rawScript\":\"print \\\"start\\\"\",\"localParams\":[],\"resourceList\":[]},\"timeout\":{},\"maxRetryTimes\":\"0\",\"taskInstancePriority\":\"MEDIUM\",\"name\":\"start\",\"dependence\":{},\"retryInterval\":\"1\",\"preTasks\":[],\"id\":\"tasks-9149\",\"workerGroup\":\"default\"},{\"conditionResult\":{\"successNode\":[\"\"],\"failedNode\":[\"\"]},\"description\":\"\",\"runFlag\":\"NORMAL\",\"type\":\"PYTHON\",\"params\":{\"rawScript\":\"print \\\"i am busy\\\"\",\"localParams\":[],\"resourceList\":[]},\"timeout\":{\"enable\":false,\"strategy\":\"\"},\"maxRetryTimes\":\"0\",\"taskInstancePriority\":\"MEDIUM\",\"name\":\"python1\",\"dependence\":{},\"retryInterval\":\"1\",\"preTasks\":[],\"id\":\"tasks-56829\",\"workerGroup\":\"default\"},{\"conditionResult\":{\"successNode\":[\"\"],\"failedNode\":[\"\"]},\"description\":\"\",\"runFlag\":\"NORMAL\",\"type\":\"PYTHON\",\"params\":{\"rawScript\":\"print \\\"end\\\"\",\"localParams\":[],\"resourceList\":[]},\"timeout\":{\"enable\":false,\"strategy\":\"\"},\"maxRetryTimes\":\"0\",\"taskInstancePriority\":\"MEDIUM\",\"name\":\"end\",\"dependence\":{},\"retryInterval\":\"1\",\"preTasks\":[],\"id\":\"tasks-45354\",\"workerGroup\":\"default\"}],\"timeout\":0}","processDefinitionLocations":"{\"tasks-9149\":{\"name\":\"start\",\"targetarr\":\"\",\"nodenumber\":\"0\",\"x\":170,\"y\":120},\"tasks-56829\":{\"name\":\"python1\",\"targetarr\":\"\",\"nodenumber\":\"0\",\"x\":170,\"y\":240},\"tasks-45354\":{\"name\":\"end\",\"targetarr\":\"\",\"nodenumber\":\"0\",\"x\":170,\"y\":360}}","processDefinitionName":"job6","projectName":"etl"}]

**Comment by hua6** [comment]
and Tasks will be executed in parallel

**Comment by xingchun-chen** [comment]
Are you using Google Chrome？

**Comment by hua6** [comment]
use google chrome

**Comment by hua6** [comment]
google chrome 83.0.4103.97（正式版本） （64 位） macbook

**Comment by xingchun-chen** [comment]
You open the incognito browser window and try to recreate the task

**Comment by hua6** [comment]
yes,use incognito browser window  will work, but why?

**Comment by xingchun-chen** [comment]
It should be caused by the browser cache, try to clear the browser cache and create a new task

**Comment by hua6** [comment]
sometimes , Sign out and Sign int twice, maybe work .but not only me, my colleague alse have this problem

**Comment by hua6** [comment]
This problem appeared after a few days

**Comment by xingchun-chen** [comment]
There can only be one line between the nodes, you have a problem with the task connection, so there will be problems with the execution of the task

**Comment by hua6** [comment]
I added these lines together，Why it is allowed to add multiple lines, maybe not just a cache problem, it should be that the front-end code lacks the necessary constraints. and after export job,"preTasks":[] is null

**Comment by xingchun-chen** [comment]
1. Under normal circumstances, the connection lines between nodes are limited
2. The preTasks of your new normal task are also null?

**Comment by break60** [comment]
> I added these lines together，Why it is allowed to add multiple lines, maybe not just a cache problem, it should be that the front-end code lacks the necessary constraints. and after export job,"preTasks":[] is null

Look at the code carefully

**Comment by hua6** [comment]
new normal task preTasks is not null, But the probability of this problem is more frequent,It's more troublesome to use privacy mode all the time

**Comment by xingchun-chen** [comment]
After clearing the cache, the problem will not occur in normal browsers, and there is no need to use incognito mode

**Comment by hua6** [comment]
I clean many times,and it will happen again ,and not only myself, my colleague alse have this problem. this version is new,  maybe other people not found now.

**Comment by break60** [comment]
> I clean many times,and it will happen again ,and not only myself, my colleague alse have this problem. this version is new, maybe other people not found now.

https://jingyan.baidu.com/article/870c6fc3466434b03fe4beea.html

**Comment by walterfung0** [comment]
> I clean many times,and it will happen again ,and not only myself, my colleague alse have this problem. this version is new, maybe other people not found now.

I meet this bug at the same version 1.3.1 as yours today. And it work after i clean chrome's cookie as suggestion. I am worried that it may become a hidden bug if we don't know the actual reason.
